### DISCUSSION TITLE
Take a look at my repository code and give me suggestions to improve it.

### DISCUSSION THREAD

**Discussion by bhos1242** [vercel/next.js#55546] [Help]
### Summary

I want to optimize my code so my website run fast i deployed it to vercel and want to add domain to it guide me

### Additional information

[_No response_
](https://github.com/bhos1242/Gurukrupa-Fire-Services)

**Comment by icyJoseph** [comment]
Is it for this one? https://github.com/bhos1242/Shri-Tushar-Ranjankar

**Comment by bhos1242** [comment]
https://github.com/bhos1242/Gurukrupa-Fire-Services   this one

**Reply by icyJoseph** [reply]
- Usage of Head in Home is not gonna work in App Router
- You have added credentials to the repository in `src/app/api/send-mail/route.js`, you should use `process.env` variables to load those
- I think in some places you can move `'use client'` directives further down the React tree
- In Next.js it is not necessary to do `import React from 'react'`, of course you NEED to `import { useState } from 'react'` to access React tools

I haven't looked much more than that. I think it was kind of alright. There's quite a lot of views and not a lot of time.
